### ISSUE TITLE
ERROR 500 for http://sprunge.us/GiCP

### ISSUE THREAD

**Issue by KinG-InFeT** [rupa/sprunge#45]
I put source from CLI: http://sprunge.us/GiCP but out error 500

**Comment by qwIvan** [comment]
Encountered the same problem

**Comment by loadaverage** [comment]
Same problem here. Maybe I can help somehow?
@rupa

**Comment by ahanjrah** [comment]
Same here, 500 Internal Server Error

**Comment by rupa** [comment]
yeah it keeps running out of resources and i'm not willing to pay too much more. need to shut it down either temporarily, to get resource usage down, or permanently, in favor of ix.io

**Comment by SoniEx2** [comment]
please shut it down temporarily, so I can download my pastes when it comes back. (if needed, please permanently disable uploads, too.)

(I'd really like to get my pastes back...)

**Comment by kevr** [comment]
@rupa can we chat about this? I'd like to help with servers if I can.

**Comment by dmd** [comment]
sprunge.us appears to be back up.
